recipes/ogdf/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "ogdf" %}
+{% set version = "2020.02" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/ogdf/ogdf/archive/catalpa-202002.tar.gz
+  sha256: 32f3f943675e55d1a23e62293d016dc1644b1176271d60f41788ade60c2d3fc5
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake >=3.1
+    - make  # [not win]
+    - ninja  # [win]
+
+test:
+  commands:
+    - test -f $PREFIX/lib/libOGDF.so  # [unix]
+    - if not exist %LIBRARY_LIB%\\libOGDF.lib exit 1  # [win]
+
+about:
+  home: http://www.ogdf.net/
+  license: GPL-2.0-only OR GPL-3.0-only
+  license_file: LICENSE.txt
```

Comment:
You need to package the license of third party source files in https://github.com/ogdf/ogdf/blob/master/LICENSE.txt

Reply:
Ok. I have included the license file in the recipe.

Reply:
It's not only the license file that is needed, but also the license files that it references.

Reply:
done

Reply:
Can they be used from the source tarball instead of copying them here?

Reply:
Please when you ask something, be specific about what you need. It's not the first time it happens. If you're not specific I will just do what I assume is correct. I spent time doing that and now I will just delete it and do something else. This is a waste of time for you and a waste of time for me.

---

Also, most of the licenses seen in https://github.com/ogdf/ogdf/blob/master/LICENSE.txt are not in the ogdf repository. I will let it as it is for now. Feel free to push what you think should be modified.

Reply:
I just assumed that you have read the conda-build docs on license_files. Please don't ask me (directly or indirectly through a team) to review any of your PRs again.